utils/plotlib/NetworkLayout.m

```diff
@@ -30,27 +30,43 @@
 		end
 		
 		function draweNBs(obj, Config)
```

Comment:
This function should be renamed imo. it does a number of things, not only drawing eNB's. Maybe something like `drawScenario`

Reply:
Maybe refactorize this into draweNB's, drawBulidings, drawCoast?

Reply:
@jakthra do you suggest to split it into 3 functions for example?

Reply:
That would be nice, but for now I think renaming the function is perfectly fine 👍